scripts/mattermost-mysql-6.0.sql

```diff
@@ -1,15 +1,19 @@
--- MySQL dump 10.13  Distrib 5.7.35, for Linux (x86_64)
+-- MariaDB dump 10.19  Distrib 10.5.10-MariaDB, for Linux (x86_64)
```

Comment:
Hmm .. why MariaDB though? Does it match with CI?

Reply:
That's just my local client, it's a comment though.

Reply:
Ah nvm, I thought we were matching the final output with this text file. Now I see that we just use this file for importing and then dump it again.

Reply:
Yeah, it's just an input "template" to initialize the migrated db.